docs/release-notes/rl-0.7.md

```diff
@@ -187,6 +188,10 @@ everyone.
 - Add [python-lsp-server](https://github.com/python-lsp/python-lsp-server) as an
   additional Python LSP server.
 
+- Add support for [edgy.nvim]. This can be enabled with
+  [](#opt-vim.ui.edgy-nvim.enable). Support for `edgy.nvim` should be considered
```

Comment:
Is this empty `[]` intended? Does it render correctly?

Reply:
It does. `[](#opt-vim...)` generates links that follow anchor titles. I've done it before, for other plugins.